application/controllers/admin/HtmlEditorPop.php

```diff
@@ -35,7 +35,11 @@ public function index()
             $aData['sControlIdDis'] = $aData['sFieldName'] . '_popupctrldis';
             $aData['toolbarname'] = 'popup';
             $aData['htmlformatoption'] = '';
-            $aData['contentsLangDirection'] = sanitize_xss_string(App()->request->getQuery('contdir'));
+            $contentsLangDirection = App()->request->getQuery('contdir');
+            if (!in_array(strtolower((string) $contentsLangDirection), ['ltr', 'rtl'])) {
+                $contentsLangDirection = null;
```

Comment:
? Then we don't have `contentsLangDirection = ;` in js ?

We don't have lang in query ?
If not : set to ltr

Reply:
Solved